shared/team-building/index.js

```diff
@@ -168,6 +191,9 @@ const styles = Styles.styleSheetCreate({
     common: {
       paddingBottom: Styles.globalMargins.small,
     },
+    isElectron: {
+      marginLeft: Styles.globalMargins.medium,
```

Comment:
shouldn't this be smaller and have a `marginRight` also to match up with the services?

Reply:
ah I see, ok